test/unit/grpcServer/handlers/tx-filter-stream/subscribeToTransactionsWithProofsHandlerFactory.spec.js

```diff
@@ -128,20 +128,6 @@ describe('subscribeToTransactionsWithProofsHandlerFactory', () => {
     }
   });
 
-  it('should respond with error if both fromBlockHash and fromBlockHeight are not specified', async () => {
```

Comment:
we need to change this spec like should respond with transactions since height 0 if both `fromBlockHash` and `fromBlockHeight` are not specified